rtmbot/core.py

```diff
@@ -61,6 +68,16 @@ def connect(self):
     def _start(self):
         self.connect()
         self.load_plugins()
+        for plugin in self.bot_plugins:
+            try:
+                self._dbg("Registering jobs for {}".format(plugin.name))
```

Comment:
I couldn't add a comment on the `_dbg` function, but what do you think about having `_dbg` log at the `DEBUG`? I guess this is more of an open question more than anything: are we going to support various debug levels?